bottlecap/src/config/env.rs

```diff
@@ -135,7 +135,7 @@ impl Default for Config {
             api_key_secret_arn: String::default(),
             kms_api_key: String::default(),
             serverless_flush_strategy: FlushStrategy::Default,
-            flush_timeout: 5,
+            flush_timeout: 90,
```

Comment:
Intentional?

Reply:
I'll move it to 10, which is the default for the go agent